content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -360,7 +360,7 @@ spec:
   ports:
   - name: http
     nodePort: 31704
-    port: 8080
+    port: 80
```

Comment:
I think you'd also need to change line 398 and https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/service/networking/nginx-secure-app.yaml for this to make sense @jeroenhartgers 

I like that the existing page makes the `port` and the `targetPort` different, because it lets you tell them apart.

Reply:
Hi @jeroenhartgers
Just checking if you are working on this PR and if you need help with the changes?
Page preview:
https://deploy-preview-24846--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/connect-applications-service/#exposing-the-service

Reply:
@sftim @kbhawkey Agreed; the fact that the targetPort should, for the sake of the example, be different. It is just historically, such as with Tomcat, more common that you would expose a (internal) target port 8080 externally to port 80 and not the other way around. I do understand that this is not a load balancer nor a nodeport. I guess we should maybe just let it be and close this PR. Thanks for your time!

Reply:
Thanks for the update @jeroenhartgers.
I will close this PR. You can reopen this pull request at any time.
/close